docs/user-guide/advanced_topics/encrypted_communication.md

```diff
@@ -0,0 +1,327 @@
+# Encrypted Communication
+
+Some applications may exchange data between federates that requires encryption in order to ensure its confidentiality.
+This could be of particular concern when the federates are communicating over the public internet between different
+institutions, and restrictions on a data set or simulator don't allow sending it over to all be run within a private
+network at a single institution.
+
+The HELICS TCP and TCPSS cores support encryption using the OpenSSL library. By default peer verification is used
+which requires both the HELICS broker and federates using encrypted communication to be set up with their own certificate
+and private key, which also allows the encrypted communication feature to act as a form of authentication between a broker
+and the federates connecting to it.
+
+This guide will provide information on building a copy of HELICS with encryption support, and show how to run a simulation
+with encrypted communication. It will not go into best practices for generating and distributing certificates or private
+keys. It is your responsibility to ensure that private keys and certificates are managed in a way that does not compromise
+the security of the encrypted communication. A good starting point for learning more about this topic is reading up on
+Public Key Infrastructure (PKI) and SSL/TLS fundamentals.
```

Comment:
Is there a link we can add here?

Reply:
There are a lot of resources that are mostly focused on how SSL applies to websites, or that are very technical; there are a few okay-ish ones that I can add links to.